src/schema/__tests__/sale_artworks.test.js

```diff
@@ -10,6 +10,44 @@ describe("Sale Artworks", () => {
     })
   }
 
+  it("pulls from /sale_artworks endpoint if `live_sale=true` and `include_lots_by_followed_artists=true`", async () => {
+    const hits = _.fill(Array(10), { id: "foo" })
+    const totalCount = hits.length * 2
+    const gravityResponse = {
+      body: hits,
+      headers: {
+        "x-total-count": totalCount,
+      },
+    }
+    const query = gql`
+      {
+        sale_artworks(
+          live_sale: true
+          include_artworks_by_followed_artists: true
+        ) {
+          counts {
+            total
+          }
+          edges {
+            node {
+              id
+            }
+          }
+        }
+      }
+    `
+
+    const { sale_artworks: { counts: { total }, edges } } = await execute(
+      gravityResponse,
+      query,
+      {
+        saleArtworksAllLoader: () => Promise.resolve(gravityResponse),
+      }
+    )
+    expect(total).toEqual(totalCount)
+    expect(edges.length).toEqual(hits.length)
+  })
+
```

Comment:
@orta - Since we're utilizing two endpoints depending on whether or not live_sale: true is passed in technically all of these tests need to be duplicated. Lets get this out to staging to test in Emission and I'll follow up with another PR.

Reply:
agree, should be pretty easy to run them twice with two types of queries at some point